x-pack/plugins/endpoint/public/applications/endpoint/store/alerts/action.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { AlertData } from '../../../../../common/types';
```

Comment:
Thoughts about setting a `baseUrl` to remove these relative imports? https://www.typescriptlang.org/docs/handbook/module-resolution.html#base-url

I'm not sure if we are going to be able to maintain our own `tsconfig.json` or if we are sharing one with the rest of Kibana.

Reply:
@alexk307 We are currently not using our own `tsconfig.json`. We could look into this, but i think it's fine for now to use relative imports. Most IDE's that provide typescript support like VSCode or Vim make relative imports pretty easy.

Reply:
FYI - I seem remember a discussion from the Kibana platform team that suggests this is something they want to look at and possibly introduce aliases that will simplify the import paths. Found discussion on Kibana issues (look at Issue 40446).